### DISCUSSION TITLE
highlighting lines on plot_parallel_coordinate

### DISCUSSION THREAD

**Discussion by ironv** [optuna/optuna#4104] [Q&A]
I use Optuna plots with `colab` backend in JupyterLab in an AzureVM and it works great.  Tooltips work perfectly for `plot_param_importances` and `plot_optimization_history`.  While `plot_parallel_coordinate` renders well and is extremely useful, it would be even more useful if a line would be highlighted by setting cursor or clicking to show all the parameter values which are part of that trial.  I downloaded the notebook and corresponding files to my local machine and it does not work there either (I have looped through all backends).  Has anybody gotten that to work?  Or is that functionality not available?  Thx.

Optuna: 3.0.2, Plotly: 5.10.0; plot.js (plotly logomark) v2.14.0

**Comment by contramundum53** [comment]
It seems that Plotly does not currently support it 😢 
https://community.plotly.com/t/is-it-possible-to-highlight-lines-and-or-add-hoverlabels-in-a-parallel-coordinates-plot/16562
